File: assets/samples/reverse.whois/classic/net/ReverseWhoisClassic.cs
using System;

public class Program
{
    static void Main(string[] args)
    {
        string username = "your whois api username";
        string password = "your whois api password";
        string term = "wikimedia";
        string url = "http://www.whoisxmlapi.com/reverse-whois-api/search.php?"
            + "term1=" + term + "&username=" + username + "&password="
            + password + "&outputFormat=JSON" + "&mode=preview";

        dynamic result = new System.Net.WebClient().DownloadString(url);
        try {
            Console.WriteLine(result);
        }
        catch (Exception e) { }
    }
}

File: assets/samples/reverse.whois/classic/java/ReverseWhoisClassic.java
public class ReverseWhoisClassic {
        public static void main(String[]args){
            String API_URL = 
                "https://www.whoisxmlapi.com/reverse-whois-api/search.php";
            String term = "wikimedia";
            String username="your whois api username";
            String password="your whois api password";
            String url = API_URL + "?mode=preview"
                + "&term1=" + term + "&outputFormat=json"
                + "&username=" + username + "&password=" + password;
            try (java.util.Scanner s = 
                new java.util.Scanner(new java.net.URL(url).openStream())) {
                System.out.println(s.useDelimiter("\\A").next());

            } catch (Exception ex) {
                ex.printStackTrace();
            }
        }
}